packages/component-library/stories/Modal.stories.tsx

```diff
@@ -47,7 +47,7 @@ export default {
 }
 
 export const ConfirmationPositiveKaizenSiteDemo = () => (
-  <ModalStateContainer isInitiallyOpen={true}>
+  <ModalStateContainer isInitiallyOpen={false}>
```

Comment:
These were intentionally open to make it easier to view the modals without having to open them. Personally I'm fine with it as I think it more accurately represents the standard modal bevahiour, but it does make it annoying if you want to quickly check the design of the opened modal. Worth checking with @didoesdigital . We had a discussion about this recently. If you look at the Elm ones, there are some stories that have "user initiated" in the name, and some that are "open by default".

Reply:
I can fix this up with a bit of refactoring. There's an experimental feature that will help us. Slot functions https://github.com/storybookjs/storybook/blob/871a4ed7bacf546f17379451a9449d6a16d3b3ff/addons/docs/docs/docspage.md#slot-functions

Reply:
Cool!

Reply:
Just spent the last little while looking into this further. It's possible, but we would kinda need to maintain two separate stories, one for the story panel that defaults to being open, and another for the docs panel that defaults to closed. The code looked like it would be confusing for someone who would be adding new stories. 

WDYT if we leave this off, and look into alternatives if we get a feature request?

Reply:
Yeah, sounds like it's not worth the complexity and maintenance burden. Sounds good to me - I think the trade-off of having this new feature is totally worth it!